src/server/game/Server/Packets/MiscPackets.h

```diff
@@ -157,6 +157,17 @@ namespace WorldPackets
             uint32 MovieID = 0;
         };
 
+        class DeathReleaseLoc : public ServerPacket
+        {
+        public:
+            DeathReleaseLoc() : ServerPacket(SMSG_DEATH_RELEASE_LOC, 4 + 4) { }
```

Comment:
When copypasting master branch code, at least dont make mistakes in it... this was `4 + (3 * 4)`